crates/ott-balancer-bin/src/balancer.rs

```diff
@@ -343,12 +345,15 @@ impl BalancerContext {
             match locator.load_epoch().cmp(&load_epoch) {
                 std::cmp::Ordering::Less => {
                     // we already have an older version of this room
+                    self.unload_room(monolith_id, metadata.name.clone()).await?;
                     return Err(anyhow::anyhow!("room already loaded"));
                 }
                 std::cmp::Ordering::Greater => {
```

Comment:
are rooms ordered by id?

Reply:
No, this code compares the `load_epoch` of the room. Only the room with the lowest `load_epoch` should be loaded, as specified in our spec doc.